static/templates/items/rules/aura.hbs

```diff
@@ -0,0 +1,210 @@
+            </div>
+
+            <div class="two-items">
+                <label>{{ localize fields.appearance.fields.highlight.fields.alpha.label }}</label>
+                {{rangePicker name=(concat "system.rules." index ".appearance.highlight.alpha") value=object.appearance.highlight.alpha placeholder="0.75" min="0" max="1" step="0.05"}}
+            </div>
+        </div>
+    </fieldset>
+
+    <fieldset>
+        <legend>{{localize fields.appearance.fields.texture.label}}</legend>
+        <div class="stacked">
+            <div class="two-items">
+                <label>{{localize fields.appearance.fields.texture.fields.src.label}}</label>
+                <input type="text" name="system.rules.{{index}}.appearance.texture.src" value="{{object.appearance.texture.src}}">
+                {{filePicker target=(concat "system.rules." index ".appearance.texture.src") type="imagevideo"}}
+            </div>
+
+            {{#if object.appearance.texture.src}}
+                <div class="two-items long-label">
+                    <label>{{localize fields.appearance.fields.texture.fields.alpha.label}}</label>
+                    {{rangePicker name=(concat "system.rules." index ".appearance.texture.alpha") value=object.appearance.texture.alpha placeholder="1" min="0" max="1" step="0.05"}}
+                </div>
+
+                <div class="two-items{{#if isVideoFile}} long-label{{/if}}">
+                    <label>{{localize fields.appearance.fields.texture.fields.scale.label}}</label>
+                    {{rangePicker name=(concat "system.rules." index ".appearance.texture.scale") value=object.appearance.texture.scale placeholder="1" min="0" max="5" step="0.01"}}
+                </div>
+
+                <div class="two-items translation long-label">
+                    <label>
+                        {{localize fields.appearance.fields.texture.fields.translation.label}}
+                        <i class="fas fa-info-circle small" data-tooltip={{fields.appearance.fields.texture.fields.translation.hint}}></i>
+                    </label>
+                    <input type="number" 
+                        name="system.rules.{{index}}.appearance.texture.translation.x"
+                        data-translation="x"
+                        placeholder="{{localize fields.appearance.fields.texture.fields.translation.fields.x.label}}"
+                        value="{{object.appearance.texture.translation.x}}"
+                    >
+                    <input type="number"
+                        name="system.rules.{{index}}.appearance.texture.translation.y"
+                        data-translation="y"
+                        placeholder="{{localize fields.appearance.fields.texture.fields.translation.fields.y.label}}"
+                        value="{{object.appearance.texture.translation.y}}"
+                    >
+                </div>
+
+                {{#if isVideoFile}}
```

Comment:
This should be a check that it's not an image file path since it's injectable. Do also remove the values or at least reset them to defaults when they're not applicable.